dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgCompanyDataTable.vue

```diff
@@ -41,28 +41,28 @@
         :key="reportingPeriod.dataId"
       >
```

Comment:
Based on PrimeVue (https://primevue.org/datatable/) the `<DataTable> component does not have an attribute `responsiveLayout`.  Why is it set to "scroll" then?  Do I overlook smth. maybe?

Reply:
I've found that attribute in two more components. Do you know why?

Reply:
It seems to have been removed with a primevue update, the IDE thinks this exists, but there is no such prop in the code or in the docs anymore. Removing it hasn't changed anything. Therefore I suggest to remove it.

Reply:
It is still there for `FrameworkDataSearchResults.vue`

Reply:
Removed it

Reply:
👍